### DISCUSSION TITLE
How do you apply for student discount?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#40268] [GitHub Education]
I entered the information shown below. 
Then I click "continue", and it asks me for my location. 
I click "accept", and then... nothing.

How do I actually apply (and get) student benefits?

<img width="729" alt="CleanShot 2022-11-28 at 14 18 57@2x" src="https://user-images.githubusercontent.com/119318204/204287524-832ab53d-20bb-432f-96cf-525d21f78610.png">

**Comment by DrSpaniel** [comment]
I had the same issue. just close the tab and retry again. worked for me

**Reply by ghost** [reply]
Unfortunately I have already tried that multiple times, and from multiple browsers, with every browser giving the exact same result..

**Comment by CrohnEngineer** [comment]
Same happening for me too! Hope somebody from @github-staff can provide some answers on this behaviour.

**Reply by ghost** [reply]
It's been 3 weeks. Any suggestions? Anybody working for GitHub? No?

**Reply by CrohnEngineer** [reply]
Still, the same issue is happening to me. Moreover, the website is asking me for location permissions even though I granted them. 
<img width="642" alt="image" src="https://user-images.githubusercontent.com/16773532/210218544-939f3b78-9708-42bf-8307-803a11942f46.png">
I see similar issues around Github Education (like #42968).  
It's really strange that nobody from @github-staff is not working or is not even aware of this issue.
